mat/cholesky.go

```diff
@@ -58,6 +71,48 @@ func (c *Cholesky) updateCond(norm float64) {
 	c.cond = 1 / v
 }
 
+// Dims returns the dimensions of the matrix.
+func (ch *Cholesky) Dims() (r, c int) {
```

Comment:
Is it worth changing the rest of the receivers to `ch`?

Reply:
Maybe in a different PR? I honestly don't care that much about receiver names for shorter methods. I was happy with `chol` everywhere but golint wasn't, so *shrug*

Reply:
Another PR is fine.

Reply:
BTW The reason `golint` was unhappy was the consistency, not the spelling. `chol` would be fine if you think that's better.